rotkehlchen/tests/utils/args.py

```diff
@@ -1,32 +1,54 @@
 from collections import namedtuple
 from typing import Optional
 
-from rotkehlchen.args import (
+from rotkehlchen.constants.misc import (
     DEFAULT_MAX_LOG_BACKUP_FILES,
     DEFAULT_MAX_LOG_SIZE_IN_MB,
     DEFAULT_SQL_VM_INSTRUCTIONS_CB,
 )
 
+ConfigurationArgs = namedtuple('ConfigurationArgs', [
```

Comment:
This is the old way of defining a named tuple. I know you just copied it, but when you touch code you should always strive to improve it!!!